packages/frontend/themes/_themebase.scss

```
@@ -27,7 +27,8 @@ $hover-contrast-change: 2%;
   --buttonPrimaryText: #{$textNavBar};
```

```suggestion
  --buttonPrimaryText: #{$textPrimary};
```

The same adjustment as for the danger button. This is `#fff`.

---

```suggestion
  --buttonPrimaryText: #fff;
```

same as for "danger"

---

nitpick: I'd appreciate the use of theming vars for all colours if possible

---

I agree but this should be solved in a  separate issue. The naming is inconsistent:

 --buttonText: #{$colorPrimary};

 --buttonPrimaryBackground: #{$colorPrimary};
 --buttonPrimaryText: #{$textNavBar};
 
  --buttonSecondaryBackground: #{$bgSecondary};
  --buttonSecondaryText: #{$textSecondary};

I introduced new btn colors as theme variables now as a first step.

Looking at the themes:
what's the reason for having these "prototype" themes?